bedrock/firefox/views.py

```diff
@@ -541,6 +542,17 @@ def get_template_names(self):
         return [template]
 
 
+def fxos(request):
```

Comment:
Looks like maybe we do need some waffle if we want the old page to show for `en-US` as well until Sunday. Here's my suggestion for the view:

```python
def firefox_os_index(request):
    locale = l10n_utils.get_locale(request)
    old_home = 'firefox/os/index.html'
    new_home = 'firefox/os/index-new.html'

    if waffle.switch_is_active('mwc-2015') and l10n_utils.template_is_active(new_home, locale):
        template = new_home
    else:
        template = old_home

    return l10n_utils.render(request, template)
```

Using whatever the actual waffle switch name is.